src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.msbuild

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@davidsh. Makes sense.  Is .msbuild the right extension for this file?  It seems very arbitrary.  @ericstj  is there a standard naming pattern for this kind of included projects?